docs/platform-engineers/system-operation/offline-installation.md

```diff
@@ -0,0 +1,60 @@
+# KubeVela Offline Installation Guide
+
+KubeVela offline installation includes the installation of KubeVela core and KubeVela Addon.
+
+## KubeVela Core offline installation
+
+### KubeVela chart
+
+- Download Helm Chart package of vela-core
+
+Download `vela-core` Helm Chart package per [Install KubeVela Core](../../install) and unarchive it.
+
+- Modify images which can be configurable
+
+Pull the following images and push them into an image repository in the dedicated environment. Overwrite each image's
+information with `-set` when installing by `helm install`.
+
+```shell
+$  kubevela git:(master) grep -r repository charts/vela-core/values.yaml -A 1
+charts/vela-core/values.yaml:  repository: oamdev/vela-core
+charts/vela-core/values.yaml-  tag: latest
+--
+charts/vela-core/values.yaml:      repository: oamdev/kube-webhook-certgen
+charts/vela-core/values.yaml-      tag: v2.3
+--
+charts/vela-core/values.yaml:      repository: oamdev/cluster-gateway
+charts/vela-core/values.yaml-      tag: v1.1.7
+--
+charts/vela-core/values.yaml:    repository: oamdev/hello-world
+charts/vela-core/values.yaml-    tag: v1
+--
+charts/vela-core/values.yaml:    repository: oamdev/alpine-k8s
+charts/vela-core/values.yaml-    tag: 1.18.2
+```
+
+- Modify images which could not be configurable
+
+The value `enableFluxcdAddon` indicates whether to enable Addon `FluxCD` by default. If it is `true`, the following images
+have to be pulled and pushed into an image repository in the dedicated environment in advance. Overwrite each image's information
+in the following YAML files.
+
+```shell
+$  kubevela git:(master) grep -r -i image: charts/vela-core/templates/addon
+charts/vela-core/templates/addon/fluxcd.yaml:                      image: fluxcd/helm-controller:v0.11.1
+charts/vela-core/templates/addon/fluxcd.yaml:                      image: fluxcd/image-automation-controller:v0.14.0
+charts/vela-core/templates/addon/fluxcd.yaml:                      image: fluxcd/image-reflector-controller:v0.11.0
+charts/vela-core/templates/addon/fluxcd.yaml:                      image: fluxcd/kustomize-controller:v0.13.1
+charts/vela-core/templates/addon/fluxcd.yaml:                      image: fluxcd/source-controller:v0.15.3
+
+
+$  kubevela git:(master) grep -r -i image: charts/vela-core  --exclude-dir=charts/vela-core/templates/addon | grep -v .Values
+charts/vela-core/templates/defwithtemplate/nocalhost.yaml:        						image: "nocalhost-docker.pkg.coding.net/nocalhost/dev-images/golang:latest"
+```
+
+- Repackage vela-core Helm Chart
```

Comment:
LGTM, it's maybe more friendly to the person first install KubeVela, if we add some example of helm package/install command line.